commitlint/plugins.ts

```diff
@@ -262,6 +262,44 @@ export abstract class Plugins {
         ];
     }
 
+    public static defaultRevertMessage(
+        headerStr: string,
+        bodyStr: string | null,
+        when = "never"
```

Comment:
@realmarv do we need a default value for when?

Reply:
I was following the commitlint rule functions. Removed it